Scripts/Editor/Utils/CodeGenerationUtility.cs

```diff
@@ -203,11 +212,25 @@ public static void GenerateStaticCollectionScript(ScriptableObjectCollection col
                 directives.Add(collection.GetType().Namespace);
                 directives.Add(typeof(List<>).Namespace);
                 directives.Add("System.Linq");
+                directives.Add("System");
                 directives.AddRange(GetCollectionDirectives(collection));
                 string className = collection.GetItemType().Name;
+
+                if (className.Contains(ReferenceConverter.BaseClassNamePostFix) && collection.Items.Count(item => item.IsReference()) > 0)
```

Comment:
I'm having some errors here, looks like this BaseClassNamePostFix is not available?
![image](https://user-images.githubusercontent.com/600419/167485804-ed99fd88-8bbe-4d96-8d01-6552a97ff9f3.png)


Testing on Unity 2020.3.23f

Reply:
That's weird because it is defined. Maybe the compiler was confused for some odd reason but I'll take a look at it on that Unity editor version.

Reply:
Ok never mind I found out the cause. I was testing it with Addressables (so the scripting define SOC_ADDRESSABLES was defined). Without it, the class `ReferenceConverter` isn't defined neither its property `BaseClassNamePostFix`.